test/e2e/.prettierrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
@rtorrero Why is adding this files as new? They were already in the project.
I think something strange happened during the rebase